.env

```diff
@@ -0,0 +1,3 @@
+# development
+NEXT_PUBLIC_API_URL="http://localhost:1337/api"
```

Comment:
.env pushed up to repo? wilddddd

Reply:
lmfao bey who set up the gitignore?????

Reply:
NextJS

Reply:
Somehow it didn't have an entry for .env files in there. I've taken the liberty of adding it oh great senior SIR! i apologize for this inconvenience and hope to be half the developer you are one day @Stanico-Knowles